filewatch/handler.go

```diff
@@ -139,6 +151,14 @@ func (lh *LogHandler) HandleLog(b []byte, catchts time.Time) error {
 			return nil
 		}
 	}
+
+	bString := string(b)
```

Comment:
please break this out into a stand alone object with the other ignore prefix thing included.

SO you can say something like `ignorer.ShouldIgnore` rather than lines and lines of loops and stuff.